compiler/z/codegen/OMRLinkage.hpp

```diff
@@ -253,10 +233,10 @@ enum TR_DispatchType
 
    Linkage(TR::CodeGenerator *);
 
-   Linkage(TR::CodeGenerator *, TR_S390LinkageConventions, TR_LinkageConventions);
+   Linkage(TR::CodeGenerator *, TR_LinkageConventions);
 
-   TR_S390LinkageConventions getExplicitLinkageType() { return _explicitLinkageType; }
-   void setExplicitLinkageType(TR_S390LinkageConventions lc ) { _explicitLinkageType = lc; }
+   TR_LinkageConventions getExplicitLinkageType() { return _explicitLinkageType; }
+   void setExplicitLinkageType(TR_LinkageConventions lc ) { _explicitLinkageType = lc; }
```

Comment:
The term "explicit" no longer has any meaning here. We should just rename all these APIs/fields and remove the word "explicit" to simplify things.

Reply:
5edbde422be4fb7741284d174c800dbc91b45796